Assets/MixedRealityToolkit/Inspectors/Utilities/MixedRealitySearchUtility.SearchConfig.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor.Search
+{
+    /// <summary>
+    /// Struct for configuring a search.
```

Comment:
Again better comments and comments for related items. How is this data container supposed to be used and what is it for?